### PR TITLE
[DEV BOARD] JOHN JOSEPH MENDEZ

### PR DESCRIPTION
**Pull request by jaymendez**
I have read and verified the following upon opening this pull request to add my information to the ReactJS Philippines Dev Board:

* [x]  My `avatar` is a square image
* [x]  My `avatar` is below 80 kB
* [x]  My `blurb` is no longer than 140 characters
* [x]  I understand that only the first 5 `skills` I list will be shown on the site

I accept any terms and conditions required for this pull request. 

Thank you!

### CODE REVIEW

File: content/devs.json [hunk 1]
```diff
@@ -183,5 +183,38 @@
...
+    "skills": [
+      "React",
+      "Typescript",
+      "GraphQL",
+      "Arduino",
+      "React Native",
+      "Python",
+      "php",
+      "ApolloGQL",
+      "Javascript",
```
Comment (jromest) [inline content/devs.json:202]: Please remove the trailing comma.

File: content/devs.json [hunk 2]
```diff
@@ -183,5 +183,38 @@
         "url": "mailto:tj.andoy@gmail.com"
       }
     ]
+  },
+  {
+    "avatar": "https://i.ibb.co/z2CwYYC/Resized-Jay.jpg",
```
Comment (jromest) [inline content/devs.json:188]: Can you scale your avatar to square? It's 757x628.

**Review by jromest — CHANGES_REQUESTED** [review]
Hey @jaymendez, thank you for adding your profile. I have some comments and request changes 😄

**Comment by jaymendez** [comment]
fixed it :)

**Review by jromest — APPROVED** [review]
Thank you @jaymendez!

**Pull request merged**